rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -51,53 +41,57 @@
 
 ipython_useSVG = False
 ipython_3d = False
```

Comment:
Yeah, the end user changes this from the notebook:
`IPythonConsole.ipython_3d = True`

Once we're comfortable with this change, we might want to think about making it the default.